lib/from_honeybee/geometry/face.rb

```diff
@@ -125,12 +125,17 @@ def to_openstudio(openstudio_model)
       # assign doors if they exist
       if @hash[:doors]
         @hash[:doors].each do |door|
-          door = Door.new(door)
-          os_subsurface_door = door.to_openstudio(openstudio_model)
-          if @hash[:face_type] == 'RoofCeiling' or @hash[:face_type] == 'Floor' && @hash[:boundary_condition][:type] == 'Outdoors'
-            os_subsurface_aperture.setSubSurfaceType('OverheadDoor')
-          end
+          honeybee_door = Door.new(door)
+          os_subsurface_door = honeybee_door.to_openstudio(openstudio_model)
           os_subsurface_door.setSurface(os_surface)
+          if door[:is_glass] == false
+            # set subsurface type
+            os_subsurface_door.setSubSurfaceType('Door')
+          elsif door[:is_glass] == true
+            os_subsurface_door.setSubSurfaceType('GlassDoor')
+          elsif @hash[:face_type] == 'RoofCeiling' or @hash[:face_type] == 'Floor' && @hash[:boundary_condition][:type] == 'Outdoors'
+            os_subsurface_door.setSubSurfaceType('OverheadDoor')
```

Comment:
This check needs to happen inside the `if door[:is_glass] == false` check. As it is written now, no door will ever get to this check and be assigned as an overhead door because it always has an `is_glass` property that is true or false. Or, if the door has no `is_glass` property, we should assume the default, which is an opaque door with `false` `is_glass`.

Also, I think you need to put parentheses around `(@hash[:face_type] == 'RoofCeiling' or @hash[:face_type] == 'Floor')`. As it's written now, it looks like an indoor `RoofCeiling` would still result in an overhead door.

Reply:
@chriswmackey I added the parentheses for `(@hash[:face_type] == 'RoofCeiling' or @hash[:face_type] == 'Floor')`. 
I added `elsif door[:is_glass] == false or door[:is_glass].nil?` after setting `OverheadDoor` to set as `Door` only if it is not `OverheadDoor` and also set `OverheadDoor` correctly even if is_glass property is nil.